### PR TITLE
Dao interface

### PR DESCRIPTION
**Pull request by yhueftle**
UPDATED AND IMPROVED!!!!!!!....!!!..!..!

### CODE REVIEW

File: src/main/java/com/ims/DAO/DAOPurchaseOrderLine.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.ims.DAO;
+
+public interface DAOPurchaseOrderLine {
+	//basic CRUD
+	void createPurchaseOrderLine();
```
Comment (cmatheny) [inline src/main/java/com/ims/DAO/DAOPurchaseOrderLine.java]: Need to take a PurchaseOrder object as input.

File: src/main/java/com/ims/DAO/DAOInventory.java [hunk 2]
```diff
@@ -3,8 +3,8 @@
 public interface DAOInventory {
 	
 	//basic CRUD
-	void createInventory();
-	void deleteInventory();
+	//void createInventory();
+	//void deleteInventory();
 	void updateInventory(int productID, int retailer);
 	
 	//Query
```
Comment (sharangw) [inline src/main/java/com/ims/DAO/DAOInventory.java]: Make query method name more specific

**Review by sharangw — CHANGES_REQUESTED** [review]
Rename query method name in DAOInventory to something more specific

**Review by cmatheny — CHANGES_REQUESTED** [review]
Throw NotYetImplementedException on non-implemented methods.

**Comment by octocat-jedi** [comment]
Can one of the admins verify this patch?

**Comment by cmatheny** [comment]
Still waiting on requested changes. Also do you have a reason for using EmptyStackException? NotYetImplementedException makes more sense to me.

**Comment by octocat-jedi** [comment]
test

**Comment by octocat-jedi** [comment]
test

**Comment by octocat-jedi** [comment]
test

**Comment by octocat-jedi** [comment]
testing

**Comment by octocat-jedi** [comment]
retest this please

**Comment by octocat-jedi** [comment]
test this please

**Review by sharangw — APPROVED** [review]

**Review by cmatheny — APPROVED** [review]

**Pull request merged**